src/Resources/contao/languages/de/tl_node.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/*
+ * Node Bundle for Contao Open Source CMS.
+ *
+ * @copyright  Copyright (c) 2019, terminal42 gmbh
+ * @author     terminal42 <https://terminal42.ch>
+ * @license    MIT
+ */
+
+$GLOBALS['TL_LANG']['tl_node']['name'] = ['Name', 'Bitte einen internen Namen des Nodes vergeben.'];
+$GLOBALS['TL_LANG']['tl_node']['type'] = ['Typ', 'Hier können Sie den Typ des Nodes auswählen.'];
+$GLOBALS['TL_LANG']['tl_node']['languages'] = ['Sprachen', 'Hier können Sprachen für den Noden eingestellt werden, nach denen im Backend gefiltert werden kann.'];
+$GLOBALS['TL_LANG']['tl_node']['tags'] = ['Tags', 'Hier können Sie Tags für den Node vergeben, nach denen Im Backend gefiltert werden kann.'];
+$GLOBALS['TL_LANG']['tl_node']['pid'] = ['Parent node'];
```

Comment:
Do we want to translate those too? :)

Reply:
I left those out deliberately, since they are just visible in the detail view and I don't like "Eltern Node". And "Übergeordneter Node" is a mouthful ;)

Reply:
Okay, I agree :)

Reply:
Also the core does not translate `pid` fields at all.